src/pages/Welcome.tsx

```diff
@@ -21,7 +21,7 @@ const Welcome = () => {
   return (
     <Box sx={styles.container}>
       <Typography variant="h1">Welcome to Chirp</Typography>
-      <img style={styles.logo} src={"/logojade.png"} alt="Logo" />
+      <img style={styles.logo} src="/chirp-logo-transparent.png" alt="Logo" />
```

Comment:
we could also implement the svg here

Reply:
Make sure to add the svg to the public folder
![chirp-logo](https://github.com/Project-Chirp/chirp-frontend/assets/73755710/a290a03f-2036-4988-9110-fee4366cad93)


```suggestion
      <Box
        component={"img"}
        style={styles.logo}
        src="/chirp-logo.svg"
        alt="Logo"
      />
```

Reply:
Thanks for making the svg, I've updated the both the logo on the navbar and the logo on the welcome page to use the new svg!